### PR TITLE
bank card: cash + discarded trains

### LINKED ISSUE #1011: Entities owned by the bank are not clearly owned by the bank

### PR DESCRIPTION
**Pull request by yzemaze**
fixes #1011

### CODE REVIEW

File: assets/app/view/game/bank.rb [hunk 1]
```diff
@@ -3,15 +3,54 @@
...
   module Game
     class Bank < Snabberb::Component
+      include Lib::Color
+
       needs :game
+      needs :layout, default: nil
 
       def render
-        props = {
+        if @layout == 'card'
```
Comment (tobymao) [inline assets/app/view/game/bank.rb:12]: layout == :card

File: assets/app/view/game/entities.rb [hunk 2]
```diff
@@ -34,7 +35,10 @@ def render
           ])
         end
 
-        children << h(:div, bank_owned.map { |c| h(Corporation, corporation: c) })
+        children << h(:div, [
+          h(Bank, game: @game, layout: :card),
+          *bank_owned.flat_map { |c| h(Corporation, corporation: c) },
```
Comment (tobymao) [inline assets/app/view/game/entities.rb:40]: should this be map?

File: assets/app/view/game/game_info.rb [hunk 3]
```diff
@@ -6,15 +6,16 @@ module View
...
 
       def render
         @depot = @game.depot
 
-        h(:div,  { style: {
-          overflow: 'auto',
-        } }, [
-          render_body,
-          ])
+        if @layout == 'discarded_trains'
```
Comment (tobymao) [inline assets/app/view/game/game_info.rb:14]: :discarded_trains

File: assets/app/view/game/entities.rb (hunk 2 above)
Comment (yzemaze) [reply assets/app/view/game/entities.rb:40]: remains from earlier dev stage, both should work

**Pull request merged**